docs/app/pages/components/sections/keyboard/hotkeys/hotkeys.component.ts

```diff
@@ -0,0 +1,123 @@
+import { ColumnSortingComponent } from './../../../../../../../src/components/column-sorting/column-sorting.component';
+import { Component, Inject, ElementRef } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import { IPlunk } from '../../../../../interfaces/IPlunk';
+import { IPlunkProvider } from './../../../../../interfaces/IPlunkProvider';
+
+@Component({
+    selector: 'uxd-components-hotkeys',
+    templateUrl: './hotkeys.component.html',
+    styleUrls: ['./hotkeys.component.less'],
+    host: {'(window:keydown)': 'hotkeys($event)'}
+})
+@DocumentationSectionComponent('ComponentsHotkeysComponent')
+export class ComponentsHotkeysComponent extends BaseDocumentationSection implements IPlunkProvider {
+
+    nativeElement: HTMLElement;
+    qButtons: HTMLElement;
+    qSelected: number = 0;
+    qText: string = 'None';
+    wButtons: HTMLElement;
+    wSelected: number = 0;
+    wText: string = 'None';
+
+    constructor( @Inject(ElementRef) private element: ElementRef) {
```

Comment:
We dont need to use the @Inject here. Angular knows what elementRef is in regards to a component so it can give it to us without using it